workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
 ---
-title: "Container Monitoring"
+title: Infrastructure Monitoring
 ---
-Overview
---------
-Container Monitoring provides real-time visibility into health and performance of containerized environments. 
 
-Container List
---------------
-The container list enables visibility into all containers across your environment excluding Kubernetes. It provides a detailed overview of your containers' health, resource consumption, and deployment. The collection of containers is container id, container name, host, status, cpu usage, ram usage, etc.
+This guide provides an overview of Middleware’s Infrastructure Monitoring page. Effortlessly collect, configure, and visualize complex telemetry data with our autodetection capabilities across your entire Infrastructure stack.
 
-![](/images/infra-monitoring/c1.png)
+Quickly isolate and drill into blockers within your teams Infrastructure stack, monitor your cluster, node, and pod data in one place, and effortlessly manage and review your overall Infrastructure utilization cluster by cluster.
 
-### Container ID
-This shows the id of the container and IDs are randomly generated and are typically 12 characters long.
+# Key Concepts
 
-### Container Name
-This shows the name of the container and the name is a user-friendly identifier that is assigned to a container.  
+### Host List
 
-### Host
-The host can be a physical or virtual machine running any operating system, such as Linux, Windows, or macOS.  
+Host List provides insight into your Kubernetes orchestration data, database usage, system-wide infrastructure utilization and more. It is the highest level compoenent overview, providing a top-down approach for diagnosing, monitoring, and maintaining your stack.
```

Comment:
Just moving things around to help the flow, and removing "kuberenetes" as this data isn't explicit to k8s. "The Host List is the highest level compoenent overview, providing a top-down approach for diagnosing, monitoring, and maintaining your stack. It provides insights into your orchestration data, database usage, system-wide infrastructure utilization and general health of your stack."